_docs/setup/upgrade.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Upgrading from Forseti v1.x
+order: 001
+---
+
+# {{ page.title }}
+
+This guide explains how to use the Forseti upgrade tool.
+
+## Before you begin
+
+Prior to upgrading the setup wizard, you will need to know:
+
+  - Forseti Security 2.0 requires G Suite enablement. 
+  You must be able to specify a G Suite Super Admin account 
+  and perform the [Domain-wide Delegation steps]({% link _docs/configure/gsuite-group-collection.md %}).
+  Have this information and be prepared to do this 
+  prior to starting the installation.
+
+  - The Forseti Security 2.0 installer only migrates configuration and rule files.
+
+
```

Comment:
We should document and set expectations around no data-migration and what will happen with existing v1.x created resources.

Reply:
Updated.